components/dashboard/src/experiments.ts

```diff
@@ -28,7 +28,7 @@ const Experiments = {
      * Experiment "example" will be activate on login for 10% of all clients.
      */
     // "example": 0.1,
-    "login-from-context-6826": 0.5, // https://github.com/gitpod-io/gitpod/issues/6826
+    "login-from-context-6826": 0, // https://github.com/gitpod-io/gitpod/issues/6826
```

Comment:
How about commenting out this line but leaving the value at 0.5?
This way

1.  (I think) new users should no longer see this particular experiment in their Localstorage - we don't want old experiments to keep accumulating.

2. The comment preserves the value used for the experiment

Reply:
FYI - I tried commenting out line 31 in 8363c65c, but that produced a typescript error
so if we did move forward with this suggestion, that would need to be fixed too.

```
/tmp/build/components-dashboard--app.d40dd25a6d6f22b76eea40734e5961a545a272a7/src/experiments.ts
TypeScript error in /tmp/build/components-dashboard--app.d40dd25a6d6f22b76eea40734e5961a545a272a7/src/experiments.ts(47,17):
Type 'boolean' is not assignable to type 'never'.  TS2322

    45 |         for (const experiment of Object.keys(Experiments) as Experiment[]) {
    46 |             if (!(experiment in result)) {
  > 47 |                 result[experiment] = Math.random() < Experiments[experiment];
       |                 ^
    48 |             }
    49 |         }
    50 |
```

Reply:
cc: @geropl for review while @laushinka is still out.

Reply:
>  I tried commenting out line 31

@jldec Yes, that's unfortunate, and rooted in my missing TS-typings skills ;-) IMO it's better to remove it (to have the compiler ensures it's not used anymore) and uncomment the `example` thing that's still in there.

Will apply that in top here, and approve afterwards if there are no objections.